prophy/composite.py

```diff
@@ -584,3 +594,12 @@ def __init__(cls, name, bases, attrs):
             extend_union_descriptor(cls, descriptor)
             add_union_properties(cls, descriptor)
         super(union_generator, cls).__init__(name, bases, attrs)
+
+
+
+class Descriptor(object):
+
```

Comment:
Consider adding `__repr__` method to make debugging simpler. Instead of:
```
(Pdb) Union.get_descriptor()[2]
<prophy.composite.Descriptor object at 0x7f1f78791c50>
```
I propose:
```
(Pdb) Union.get_descriptor()[2]
<a, INT>
```